compiler/wasm/src/compile.rs

```diff
@@ -17,6 +17,9 @@ use crate::errors::JsCompileError;
 
 const BACKEND_IDENTIFIER: &str = "acvm-backend-barretenberg";
 
+const NOIR_ARTIFACT_VERSION_STRING: &str =
```

Comment:
We can probably define this in noirc_driver, then re-export it from there to Nargo and noir_wasm